Framework/Algorithms/src/PolarizationCorrections/HeliumAnalyserEfficiency.cpp

```diff
@@ -37,10 +37,10 @@ auto constexpr OUTPUT_WORKSPACE{"OutputWorkspace"};
 auto constexpr OUTPUT_FIT_CURVES{"OutputFitCurves"};
 auto constexpr OUTPUT_FIT_PARAMS{"OutputFitParameters"};
 auto constexpr SPIN_STATES{"SpinStates"};
-auto constexpr PD{"GasPressureTimesCellLength"};
-auto constexpr PD_ERROR{"GasPressureTimesCellLengthError"};
-auto constexpr START_LAMBDA{"StartLambda"};
-auto constexpr END_LAMBDA{"EndLambda"};
+auto constexpr PXD{"PXD"};
```

Comment:
This is unrelated to the changes requested under the issue, but while we're in this part of the code, please could we change the type of all of the string constants in the `PropertyNames` namespace from `auto constexpr` to `static const std::string`?

I've been doing some more reading about this and it looks like you can end up with unintended copies if you use the `constexpr` type for string constants. I've since noticed that it seems more common to use `static const std::string` for them in Mantid.

Reply:
I’m assuming that the unintended copies might be occurring because the string constants are deduced by the compiler as `const char*`. However, in Mantid, these constants will be used as `std::string`, so the conversion from `const char*` to `std::string` could be introducing some overhead. If this isn’t the reason, I would like to know. I have refactored the code to use `static const std::string`. Thanks.

Reply:
Yes, that's the reason.